### ISSUE TITLE
Line Charts: month abbreviations are used in aria-labels instead of full month names

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/wc-admin#1659]
**To Reproduce**
1. Go to any report.
2. Navigate the line chart with a screen reader.
3. Notice the line regions are named `Previous Period (Feb 20, 2019)` but they should be `Previous Period (February 20, 2019)`.

This affects all intervals.
![image](https://user-images.githubusercontent.com/3616980/53160792-f1ebbb00-35c8-11e9-96bf-c24f0c870fbf.png)
